### DISCUSSION TITLE
Re-enable the Timeline option to Disable Quick Play

### DISCUSSION THREAD

**Discussion by audacitymike** [audacity/audacity#4565] [Ideas & Feature Requests]
Can the option to disable quick play be re-introduced soon?

**Comment by timoteo5** [comment]
PLEEEEASE!  I don't understand why such a crucial toggle would be removed.  I almost NEVER want it to auto-play when I click on the timeline ... it is not how any of my other timeline-based software works (AE, NLE's, other audio apps, 3d apps, etc.)

OR ... is there a way I am unaware of (I am pretty new to Audacity) that allows you to mark the cursor location across all tracks without playing and stopping playback as is currently required?

Thanks!

**Reply by petersampsonaudacity** [reply]
> is there a way I am unaware of (I am pretty new to Audacity) that allows you to mark the cursor location across all tracks without playing and stopping playback as is currently required?

1) You can use the "**X**" shortcut to stop play and set the cursor at the current play position, then
2) use **Ctrl+B** to set a point label at that current cursor position



> PLEEEEASE! I don't understand why such a crucial toggle would be removed. I almost NEVER want it to auto-play when I click on the timeline ... it is not how any of my other timeline-based software works (AE, NLE's, other audio apps, 3d apps, etc.)

I can't remember quite why it was decided a long time ago to remove that option - @SteveDaulton  (who wrote the Timeline Quick Play) may recall the reasoning.

It was removed fairly recently in Audacity 3.1.0 (Oct 2021) - 3.0.5 was the last Audacity version to have the option to turn TQP off.

**Reply by timoteo5** [reply]
Thanks for that info.  Unfortunately, besides being extra steps, it doesn't quite do what I want to do.  I don't want to add a marker for no reason (will clutter up the timeline) and it only does it on one track.  I need it to span tracks.

**Comment by SteveDaulton** [comment]
> I almost NEVER want it to auto-play when I click on the timeline

If you never want to auto-play when you click on the timeline, why do you click on the timeline?
If you just want to set the cursor position, click on the audio track.

**Reply by timoteo5** [reply]
Thanks, but same as above reply ... I need it to span the entire timeline, not just one track.  Also there does not appear to be anyway to SNAP-TO anything but whole seconds, right?  I don't see an option to even snap to markers, play-head position, etc.  Let me know if I am just missing something.

It's very possible I am missing some vital functionality, as I said I am new to Audacity. but out of curiosity, how do you (or anyone else who wouldn't mind commenting) line up things precisely on an Audacity timeline?  Especially when you have multiple tracks going down the page.

Thanks!

**Reply by SteveDaulton** [reply]
"SnapTo" snaps to whatever units are selected in the Selection Toolbar time controls.
Details are in the manual: https://manual.audacityteam.org/man/selection_toolbar.html

**Reply by timoteo5** [reply]
Thanks, but snapping to any time setting is not helpful.  (sorry, I wasn't clear I meant "the only option is to snap to a set time interval.")

For example, in all of the NLE's I use, you can snap to play-head position, a clip, a marker, a rubber-band point, etc., etc.

**Reply by SteveDaulton** [reply]
If you want DAW features, then perhaps Reaper would suit you better as that is a full featured DAW. Audacity is still basically an Audio editor, though it does have some DAW-like features.

**Reply by timoteo5** [reply]
I have heard of it, will definitely check it out.  Thanks, Steve.

**Comment by petersampsonaudacity** [comment]
Ah now I look at the 3.1 release notes I recall the history:
https://wiki.audacityteam.org/wiki/Release_Notes_3.1

There it says:
>Timeline Quick Play for regions and locked regions have been replaced by playback looping.

With that version Muse added a new version of looped playback which at the time they thought made TQP entirely redundant.

Many users (including me, _I use TQP a LOT_ _as Audacity's scrubbing is not usable IMVHO_) did not agree and campaigned to get TQP added back as an adjunct to new looping.  But when it was added back the former option to turn it off, if required, was apparently not added back.

**Reply by timoteo5** [reply]
Thanks for sharing which version it was last in ... any idea the biggest downside(s) to downgrading to that version are?

**Comment by Luciftian** [comment]
I also need to join the call to have the ability to disable quick play. Looping is not the same thing.
It is highly irritating and time-wasting for the track begin to autoplay every time I move the play head on the timeline.

**Comment by petersampsonaudacity** [comment]
I have marked this as a regression

as before new loping was introduced in 3.1 - not only was Timeline Quick-Play available there was also an entry in the Timeline context menu to optionally disable it.

Timeline Quick-Play was reintroduced in 3.1.0 during the alpha development, but somehow adding back the option to turn off it was overlooked.

### This is 3.0.5
![image](https://github.com/audacity/audacity/assets/50205984/3acb5384-6ebb-451e-953d-cd7eef495d2a)

### This is 3.1.0
![image](https://github.com/audacity/audacity/assets/50205984/df1e3c87-073c-4d73-bd1a-b9e61e5c7216)


@LWinterberg  as this is a regression, can we please have this converted into an **Issue** rather than just a Discussion?

**Comment by petersampsonaudacity** [comment]
@LWinterberg 

Another complaint about this on the Forum today:  

https://forum.audacityteam.org/t/need-disable-the-play-head-automatically-play-when-click-to-timeline/98775

**Comment by scottwegner** [comment]
I would also like to agree with those who have requested the ability to disable quick play.

**Comment by petersampsonaudacity** [comment]
This is fixed in the alpha test build for Audacity version 4

Quick-Play is off by default and there is an option to turn it on from the Timeline context menu:
<img width="1092" height="819" alt="image" src="https://github.com/user-attachments/assets/2e246774-932b-4043-be14-227eb200e8d6" />
